.eslintrc

```diff
@@ -15,6 +15,14 @@
     "prettier/prettier": ["error", {"printWidth": 150}]
   },
   "parserOptions": {
-   "ecmaVersion": 2017
- }
+    "ecmaVersion": 2017
+  },
+  "overrides": [
+    {
+      "files": "*.js",
+        "rules": {
+          "flowtype/require-valid-file-annotation": "off"
```

Comment:
You could just put it into the general "rules" section. eslint just applies for .js files anyways :)

Reply:
Here I was dutifully following what was in the Changelog, but yeah, that's much better :)

Reply:
Yeah in the changelog that was more targeting the situation where you want to disable it for certain files.